doc/jsk_perception/nodes/probability_classifier.rst

```diff
@@ -0,0 +1,40 @@
+probability_image_classifier.py
+===============================
+
+
```

Comment:
Please add visualized image.
Maybe you can use `draw_classification_result.py`.